### DISCUSSION TITLE
How to exclude folder "BONUS" and relative sub-folders?

### DISCUSSION THREAD

**Discussion by RealSting** [Komet/MediaElch#1729] [Q&A]
This (on advancedsettings.xml) does not work:

`<pattern applyTo="folders">.*(?i)bonus.*</pattern>`

**Comment by bugwelle** [comment]
The folder pattern is applied on _folder names_. If you want to exclude `BONUS`, this  should be enough:

```xml
<pattern applyTo="folders">BONUS</pattern>
<pattern applyTo="folders">IBONUS</pattern>
```
